docs/02-app/01-building-your-application/01-routing/03-linking-and-navigating.mdx

```diff
@@ -117,48 +124,55 @@ export default function Page() {
+>
+> - Prefetching is not enabled in development, only in production.
 
-[Server Actions](/docs/app/building-your-application/data-fetching/server-actions) can be used to revalidate data on-demand by path ([`revalidatePath`](/docs/app/api-reference/functions/revalidatePath)) or by cache tag ([`revalidateTag`](/docs/app/api-reference/functions/revalidateTag)).
+### 2. Caching
 
-### Prefetching
+Next.js has an **in-memory client-side cache** called the [Router Cache](/docs/app/building-your-application/data-fetching/caching#router-cache). As users navigate around the app, the React Server Payload of [prefetched](#prefetching) route segments and visited routes are stored in the Router Cache.
 
-Prefetching is a way to preload a route in the background before it's visited. The rendered result of prefetched routes is added to the router's client-side cache. This makes navigating to a prefetched route near-instant.
+This means on navigation, the cache is reused as much as possible, instead of making a new request to the server; improving performance by reducing the numbers of requests and data transferred.
 
-By default, routes are prefetched as they become visible in the viewport when using the `<Link>` component. This can happen when the page first loads or through scrolling. Routes can also be programmatically prefetched using the `prefetch` method of the [`useRouter()` hook](/docs/app/api-reference/functions/use-router#userouter).
+Learn how to configure the [Router Cache](/docs/app/building-your-application/data-fetching/caching).
 
-**Static and Dynamic Routes**:
+#### 3. Partial Rendering
 
-- If the route is static, all the Server Component payloads for the route segments will be prefetched.
-- If the route is dynamic, the payload from the first shared layout down until the first `loading.js` file is prefetched. This reduces the cost of prefetching the whole route dynamically and allows [instant loading states](/docs/app/building-your-application/routing/loading-ui-and-streaming#instant-loading-states) for dynamic routes.
+Partial rendering means only the route segments that change on navigation re-render on the client, and any shared segments are preserved.
 
-> **Good to know**:
->
-> - Prefetching is only enabled in production.
-> - Prefetching can be disabled by passing `prefetch={false}` to `<Link>`.
+For example, when navigating between two sibling routes, `/dashboard/settings` and `/dashboard/analytics`, the `settings` and `analytics` pages will be rendered, and the shared `dashboard` layout will be preseved.
+
+<Image
+  alt="How partial rendering works"
+  srcLight="/docs/light/partial-rendering.png"
+  srcDark="/docs/dark/partial-rendering.png"
+  width="1600"
+  height="945"
+/>
+
+Without partial rendering, each navigation would cause the full page to re-render on the server. Rendering only the segment that changes reduces the amount of data transferred and execution time, leading to improved performance.
 
 ### Soft Navigation
```

Comment:
This section needs to reflect that there is no clear separation between soft and hard navigation anymore. It might be worth outlining the 30 seconds behavior and such, or linking to the router cache doc.